File: dataset/avatar/Python/Code/codeforces_682_B.py
import os , sys , io , math
from array import array
from math import *
I = lambda : [ * map ( int , sys.stdin.readline ( ).split ( ) ) ]
IS = lambda : input ( )
IN = lambda : int ( input ( ) )
IF = lambda : float ( input ( ) )
n = IN ( )
l = I ( )
f = 1
l.sort ( )
c = 0
for i in l :
    if i > c : c += 1
print ( c + 1 )

File: output/qwen_2.5_72b_instruct/avatar/algo_based_translation/python/java/successful_translation/codeforces_682_B.java

import java.util.ArrayList;
import java.util.Collections;
import java.util.List;
import java.util.Scanner;

public class codeforces_682_B {
    private static List<Integer> I(Scanner scanner) {
        String[] input = scanner.nextLine().split(" ");
        List<Integer> result = new ArrayList<>();
        for (String s : input) {
            result.add(Integer.parseInt(s));
        }
        return result;
    }

    private static String IS(Scanner scanner) {
        return scanner.nextLine();
    }

    private static int IN(Scanner scanner) {
        return Integer.parseInt(scanner.nextLine());
    }

    private static float IF(Scanner scanner) {
        return Float.parseFloat(scanner.nextLine());
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = IN(scanner);
        List<Integer> l = I(scanner);
        int f = 1;
        Collections.sort(l);
        int c = 0;
        for (int i : l) {
            if (i > c) {
                c += 1;
            }
        }
        System.out.println(c + 1);
        scanner.close();
    }
}

